apps/api/src/http/controllers/admin/fdbs/category.controller.ts

```diff
@@ -156,6 +157,14 @@ function adminCategoryController({
     res.json(categories);
   };
 
+  const all = async (
```

Comment:
This might quickly become a perf hog and cause issues for larger sets, better use paginated query/response?

Reply:
Please correct me if I'm wrong, but it looks like this isn't being used anywhere. Also, it looks like browseMain already implements what this is intended to do but with pagination. Should it be removed?